frontend/app/components/redesign/components/landing/ToolCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { Link } from '@remix-run/react'
+import { PillTag } from './PillTag'
+import arrowOutwardIcon from '~/assets/images/landing/arrow-outward.svg'
+
+export type ToolCardProps = {
+  children?: React.ReactNode
+  title: string
+  tags: string[]
+  icon: string
+  to?: string
+  href?: string
+  target?: '_blank' | '_self' | '_parent' | '_top'
+  className?: string
+}
+
+export const ToolCard = ({
+  children,
+  title,
+  tags,
+  icon,
+  to,
+  href,
+  target = '_blank',
+  className = ''
+}: ToolCardProps) => {
+  const linkContent = title
+  const linkClasses =
+    "font-bold text-xl leading-normal text-text-primary after:absolute after:inset-0 after:content-[''] after:z-10"
```

Comment:
Tailwind adds the `content: ''` itself.
```suggestion
    "font-bold text-xl leading-normal text-text-primary after:absolute after:inset-0 after:z-10"
```